stackit/internal/services/iaas/networkinterface/resource.go

```diff
@@ -48,7 +48,6 @@ type Model struct {
 	Name               types.String `tfsdk:"name"`
 	AllowedAddresses   types.List   `tfsdk:"allowed_addresses"`
 	IPv4               types.String `tfsdk:"ipv4"`
-	IPv6               types.String `tfsdk:"ipv6"`
```

Comment:
should we remove ipv6 also from datasource

Reply:
Yes, good catch